subsys/spm/Kconfig

```diff
@@ -66,6 +76,21 @@ config SPM_SECURE_SERVICES
 	  Note: Please set this and SPM_SERVICE_* configs from the app, instead
 	  of the SPM. This ensures that the values are in sync between the two
 	  images.
+
+config SPM_NS_DEBUG_LOGS_DUMP
```

Comment:
Did you consider presenting this as two different features?
1: In SPM: A general fatal error handler calling into NS when an S error happens
2: In NS: Using the SPM feature to dump logs.
The function naming on the SPM side is already quite generic, so it seems natural.
Also, I found it a bit confusing that the kconfig is used in both SPM and NS, and it does different things in those two.

Reply:
I applied this proposal.